src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -77,8 +77,8 @@
 
```

Comment:
I have another PR that will remove NodeJSGapicContext when I migrate the grpc documentation gen.